src/interfaces/index.ts

```diff
@@ -1,31 +0,0 @@
-import ActivateCard from "./ActivateCard.js";
-import BlockOrUnlockCard from "./BlockOrUnlockCard.js";
-import Business from "./Business.js";
-import Card from "./Card.js";
-import CardMetrics from "./CardMetrics.js";
-import Company from "./Company.js";
-import Employee from "./Employee.js";
-import NewCard from "./NewCard.js";
-import Payment from "./Payment.js";
-import Recharge from "./Recharge.js";
-import VinculatedCard from "./VinculatedCard.js";
-import DeleteCard from "./DeleteCard.js";
-import POSPurchase from "./POSPurchase.js";
-import OnlinePurchase from "./OnlinePurchase.js";
-
-export {
-    ActivateCard,
-    BlockOrUnlockCard,
-    Business,
-    Card,
-    CardMetrics,
-    Company,
-    Employee,
-    NewCard,
-    Payment,
-    Recharge,
-    VinculatedCard,
-    DeleteCard,
-    POSPurchase,
-    OnlinePurchase
-};
```

Comment:
Cuidado com as interfaces muito especificas, se for o caso de ser usada em apenas um caso, poderia deixar no próprio arquivo que vai ser usada. Lembrando que pode usar o outra formas como `Omit`, veja nesse link para entender melhor outros usos:  https://www.typescriptlang.org/docs/handbook/utility-types.html